rules/saiserver.mk

```diff
@@ -0,0 +1,7 @@
+# saiserver
+
+SAISERVER = saiserver
```

Comment:
I feel it worth to introduce flexibility so that we allow to embed binary into the image directly?